pkgs/applications/misc/mucommander/default.nix

```diff
@@ -1,21 +1,23 @@
-{ stdenv, fetchFromGitHub, gradle_3_5, perl, makeWrapper, jre, gsettings-desktop-schemas }:
+{ stdenv, fetchFromGitHub, gradle_4_10, perl, makeWrapper, jre, gsettings-desktop-schemas }:
 
 let
-  version = "0.9.2";
-  name = "mucommander-${version}";
+  version = "0.9.3";
+  patch = "3";
+  name = "mucommander-${version}-${patch}";
 
   src = fetchFromGitHub {
     owner = "mucommander";
     repo = "mucommander";
-    rev = version;
-    sha256 = "1fvij0yjjz56hsyddznx7mdgq1zm25fkng3axl03iyrij976z7b8";
+    rev = "${version}-${patch}";
+    sha256 = "13yks3lalla4jpbl9qnvi4swmp049yibga1kmqpbjda08adi4cn1";
```

Comment:
I got the a different checksum:


```
hash mismatch in fixed-output derivation '/nix/store/ls186qmbqva0jkxq2p8xclwhgbl3fa35-source':
  wanted: sha256:13yks3lalla4jpbl9qnvi4swmp049yibga1kmqpbjda08adi4cn1
  got:    sha256:1zhglsx3b5k6np3ppfkkrqz9wg0j7ip598xxfgn75gjl020w0can
```

Reply:
Yes, you're right. I was tripped by the fact that I had used tag 0.9.3 before realizing they released patch versions, like 0.9.3-1 and so forth.

Also, they released 0.9.3-3, but the version inside the build.gradle file says 0.9.4... yuck.

I'll fix this.